### PR TITLE
Add PHP 7.2.6 runtime

### PR DESCRIPTION
**Pull request by akrabat**
This runtime uses an updated router.php which runs the action code in
the same process and enables PHP's OPcache byte-code cache. We also
read the /init config information using `require` rather than parsing a
JSON file.

This results in a significant performance improvement as can be seen
in the typical gatlingRun-LatencySimulation results on my dev machine.
The mean response times are:

* Warm php:7.1 invocation  : 70ms
* Warm nodejs:8 invocation : 19ms
* Warm php:7.2 invocation  : 19ms

The change to router.php can also be applied to the PHP 7.1 action and it will see similar performance improvements.

### CODE REVIEW

File: core/php7.2Action/php.ini [hunk 1]
```diff
@@ -0,0 +1,1948 @@
+; Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (csantanapr) [inline core/php7.2Action/php.ini]: this file looks too big and boiler plate.
Can we have a short version with a few settings that we need to override

**Comment by csantanapr** [comment]
The new test file should look like this if there it's the same set of tests as 7.1
```scala
package runtime.actionContainers

import org.junit.runner.RunWith
import org.scalatest.junit.JUnitRunner

@RunWith(classOf[JUnitRunner])
class Php72ActionContainerTests extends Php7ActionContainerTests {

  override lazy val nodejsContainerImageName = "action-php-v7.2"
}
```

File: core/php7.2Action/php.ini (hunk 1 above)
Comment (akrabat) [reply core/php7.2Action/php.ini]: Fixed.

**Comment by akrabat** [comment]
@csantanapr Tests added now!

**Pull request merged**